deploy/kubernetes/base/controller.yaml

```diff
@@ -51,3 +51,4 @@ spec:
         - name: cloud-sa-volume
           secret:
             secretName: cloud-sa
+  volumeClaimTemplates: []
```

Comment:
Why is this needed?

Reply:
It's a bug in kustomize, will add a comment: https://github.com/kubernetes-sigs/kustomize/issues/504